### PR TITLE
chore(main): add documentation for Windows (WSL2) installation

### PR DESCRIPTION
**Pull request by namwoam**
Because

- We want to add support `instill-core` on Windows.

This commit

- Provides documentation for installing `instill-core` on Windows through WSL2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,17 @@ Not quite into self-hosting? We've got you covered with **☁️ [Instill Cloud]
...
+
+- **Windows** - **🔮 Instill Core** works on Windows through Windows Linux Subsystem (WSL2)
+  
+  *WSL specific settings*:
+  
+  (a) The package `yq` is not installed on Ubuntu WSL2 by default, install the lastest version of `yq` from the [official Github Repository](https://github.com/mikefarah/yq).
+  
+  (b) Install the latest version of Docker Desktop on Windows and enable the WSL2 integration following [this tutorial from Microsoft](https://docs.docker.com/desktop/wsl/)
+  
+  (c) Install `cuda-toolkit` on WSL2 following [this tutorial from Nvidia](https://docs.nvidia.com/cuda/wsl-user-guide/index.html#getting-started-with-cuda-on-wsl-2)
```
Comment (pinglin) [inline README.md:72]:
```suggestion
  - Install the lastest version of `yq` from the GitHub [Repository](https://github.com/mikefarah/yq), as the package `yq` is not installed on Ubuntu WSL2 by default
  
  - Install the latest version of Docker Desktop on Windows and enable the WSL2 integration following the [tutorial](https://docs.docker.com/desktop/wsl) by Docker
  
  - (optional) Install `cuda-toolkit` on WSL2 following the [tutorial](https://docs.nvidia.com/cuda/wsl-user-guide/index.html#getting-started-with-cuda-on-wsl-2) by NVIDIA
```

**Review by pinglin — CHANGES_REQUESTED** [review]
Please change as requested to simplify wording. Thanks.

File: README.md (hunk 1 above)
Comment (namwoam) [reply README.md]: Changed as requested.

**Review by pinglin — APPROVED** [review]
LGTM

**Pull request merged**